README.md

```diff
@@ -5,6 +5,8 @@ Welcome to the home page for the ICU4X-SC.  We are a subcommittee of ICU-TC in t
 
 Please subscribe to this repository to participate in discussions.  If you would like to join the core development team, apply to join [our mailing list](https://groups.google.com/a/unicode.org/forum/#!forum/omnicu-core).
 
+![Rust](https://github.com/echeran/icu4x/workflows/Rust/badge.svg?branch=master)
```

Comment:
Where does the string `Rust` come from in the URL?

Reply:
Changed accordingly, thanks.  I started off with the [Rust workflow](https://github.com/actions/starter-workflows/blob/master/ci/rust.yml) in the official [starter workflows](https://github.com/actions/starter-workflows) repo, which was called "Rust", which is where the string comes from.  I subsequently changed the name of the workflow, and the old workflow name seemed to redirect to the new workflow, just like when renaming a repo.